src/__tests__/unit/lib/e-mem/index.test.ts

```diff
@@ -0,0 +1,185 @@
+import {EMemModel} from '../../../../lib';
+//import { ERRORS } from '../../../../util/errors';
+//const { InputValidationError } = ERRORS;
```

Comment:
you can remove imports if not used

Reply:
![spv-bitcoin.png](https://github.com/Green-Software-Foundation/if-models/assets/147960298/1ef24149-1724-4577-81e7-1a2fc433d8f3)